Content/0.4 - Settings/settings.json

```diff
@@ -10,7 +10,9 @@
 	"settings_importFromLegacyWallet": "Import from a Legacy Wallet",
 	"settings_ledgerHardwareWallets": "Ledger Hardware Wallets",
 	"settings_depositGuarantees": "Deposit Guarantees",
+	"settings_depositGuarantees_title": "Deposit Guarantees",
 	"settings_depositGuarantees_subtitle": "Set your default guaranteed minimum for estimated deposits",
+	"settings_depositGuarantees_text": "Set the guaranteed minimum deposit to be applied whenever a deposit in a transaction can only be estimated.\n\nYou can always change the guarantee from this default in each transaction.",
```

Comment:
Where this is supposed to be shown ?

Reply:
This is the main text for the deposit guarantees screen in Settings -> Account Security & Settings -> Deposit Guarantees.
<img width="200" alt="image" src="https://github.com/radixdlt/apps-localization-src/assets/123396602/93aa6496-7144-42e0-a88f-f554ecf952f9">

Reply:
Ah makes sense yes 👍